src/plugins/mf_priority.cpp

```diff
@@ -812,6 +812,15 @@ extern "C" int flux_plugin_init (flux_plugin_t *p)
         || flux_jobtap_service_register (p, "reprioritize", reprior_cb, p)
         || flux_jobtap_service_register (p, "rec_q_update", rec_q_cb, p) < 0)
         return -1;
+
+    struct queue_info *q;
+    q = &queues["default"];
+
+    q->min_nodes_per_job = 0;
+    q->max_nodes_per_job = 1024;
+    q->max_time_per_job = 64000;
+    q->priority = 1000;
```

Comment:
Question: since queues are not validated, these numbers don't matter right?

If they do, I'd suggest the equivalent of unlimited for `max_time_per_job` and `max_nodes_per_job`.

Reply:
Good question! `min_nodes_per_job`, `max_nodes_per_job`, and `max_time_per_job` are not used or enforced anywhere throughout the plugin, so those values should not matter. `priority` is the only factor used in the priority calculation for jobs.

Reply:
I'd add a comment here to that effect then, to avoid confusion for someone browsing the code.

Reply:
Sounds good. I just force-pushed a fix to the first commit that adds a comment to make note that those three values aren't currently used or enforced in the plugin.